test/report_test.rb

```diff
@@ -31,7 +32,7 @@ def test_announce_with_comment
       ANNOUNCEMENT
 
       report = Report.new(repo, comment: "Only include new features") # Not sure how to test the contents of the editor
-      announcement = report.announce
+      announcement = report.announce(team: 'bobcats', token: '1234')
```

Comment:
Something about having to add these every time we `announce` feels a little off (especially since the spec doesn't care about `bobcats` or `1234 🤔). Is there perhaps a way we can preserve the previous interface?

Reply:
The interface hasn't been changed. We didn't have to specify them before because the slackistrano gem was not installed when running unit tests and `SlackDummy` didn't need any options.